scripts/travis/run_test.sh

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
+if [[ $TRAVIS_OS_NAME == 'osx' ]]; then
```

Comment:
BTW, is it possible to do this if in travis instead of a bash script? E.g.:

```
- os: osx
  script:
    - scripts/travis/osx/build.sh
- os: linux
  script:
    - scripts/travis/linux/docker.sh rocproject/ci-ubuntu:17.04 
    ...
```

Reply:
Fixed